README.md

```diff
@@ -1,41 +1,142 @@
-# npe2
+# npe2 - napari plugin engine version 2
 
 [![CI](https://github.com/napari/npe2/actions/workflows/ci.yml/badge.svg)](https://github.com/napari/npe2/actions/workflows/ci.yml)
 [![codecov](https://codecov.io/gh/napari/npe2/branch/main/graph/badge.svg?token=FTH635x542)](https://codecov.io/gh/napari/npe2)
 
-napari plugin refactor
+## Project description
 
-see also https://github.com/napari/napari/issues/3115
+The **napari plugin engine version 2**, **npe2** extends the functionality of
+[napari's core](https://github.com/napari/napari).
+The plugin ecosystem offers user additional functionality for napari as well
+as specific support for different scientific domains.
 
-## Documentation
+This repo contains source code and documentation about the napari plugin engine.
 
-For documentation on authoring npe2 plugins, see the [napari plugin docs](https://napari.org/stable/plugins/index.html).  These include:
-- the [manifest reference](https://napari.org/stable/plugins/technical_references/manifest.html)
-- the [contribution guide](https://napari.org/stable/plugins/building_a_plugin/guides.html)
+## Getting started
 
-## Command line tool
+The [napari plugin docs landing page](https://napari.org/stable/plugins/index.html)
+offers comprehensive information for **plugin users** and for **plugin developers**.
+
+### Plugin users
+
+For plugin users, the docs include information about:
+- [Starting to use plugins](https://napari.org/stable/plugins/start_using_plugins/index.html#plugins-getting-started)
+- [Finding and installing plugins](https://napari.org/stable/plugins/start_using_plugins/finding_and_installing_plugins.html#find-and-install-plugins)
+
+### Plugin developers
+
+For plugin developers, the docs cover topics like:
+- [Building a plugin](https://napari.org/stable/plugins/building_a_plugin/index.html)
+- [Contributing a plugin](https://napari.org/stable/plugins/building_a_plugin/guides.html)
+- [Technical references such as the plugin manifest](https://napari.org/stable/plugins/technical_references/manifest.html)
+
+Try the [**napari plugin template**](https://github.com/napari/napari-plugin-template)
+to streamline development of a new plugin.
+
+## Installation
+
+The `npe2` command line tool can be installed with `pip` or `conda`, but will already be installed as a dependency if you have napari installed.
+
+### Using pip
+
+1. Create and activate a virtual environment.
+
+    ```bash
```

Comment:
Do we think it's better to link here to our virtual environments guide: https://napari.org/stable/plugins/virtual_environment_docs/1-virtual-environments.html

I can see how we might just want a quickstart so feel free to take or leave this suggestion!

Reply:
Let's do this. Let's add the documentation link below the quickstart commands.

In general, I like to keep all installation steps together instead of linking out to other docs. Installation is the first experience users have with the repo, and we want them to be successful.

Reply:
Added an italicized note to with the suggested guide. 👍🏼